cmd/eksctl-anywhere/cmd/flags.go

```diff
@@ -0,0 +1,47 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+type clusterCmdOptions struct {
+	clusterOptions
+	forceClean bool
```

Comment:
```suggestion
	forceCleanup bool
```

Reply:
Thanks Taylor, updated forceCleanup field name and made it part of clusterOptions

Reply:
Decided to not group the flags in a separate function and not use this struct. Ended up not touching forceCleanup.